cfme/infrastructure/resource_pool.py

```diff
@@ -83,7 +80,7 @@ def get_detail(self, *ident):
         Returns: A string representing the contents of the InfoBlock's value.
         """
         if not self._on_detail_page():
-            sel.force_navigate('infrastructure_resource_pool', context=self._get_context())
+            navigate_to(self, 'Details')
```

Comment:
So what I did here in a previous PR is to put the ```self._on_detail_page()``` detection into an ```am_i_here``` method, which does the job of removing a method AND making the Detail's page ```am_i_here```aware.